### PR TITLE
Added support for events

### PR DESCRIPTION
**Pull request by FelGel**

### CODE REVIEW

File: pkg/controlplane/http/session.go [hunk 1]
```diff
@@ -55,24 +55,34 @@ func NewSession(parentLogger logger.Logger,
...
-	// try to create the resource
-	err := newSession.createResource(newSessionInput.Ctx,
-		"sessions",
-		"session",
-		&newSessionInput.ControlPlaneInput,
-		&newSessionInput.SessionAttributes,
-		&output,
-		&responseSessionAttributes)
+		// Generate cookie from access key
+		newSession.cookies["session"] = fmt.Sprintf("session=j%%3A%%7B%%22sid%%22%%3A%%20%%22%s%%22%%7D", newSessionInput.AccessKey)
```
Comment (pavius) [inline pkg/controlplane/http/session.go]: Isn't there a programmatic way to generate this string?

File: pkg/controlplane/http/session.go [hunk 2]
```diff
@@ -168,6 +178,22 @@ func (s *session) UpdateClusterInfoSync(
 	return &updateClusterInfoOutput, nil
 }
 
+// CreateEvent emits an event
+func (s *session) CreateEventSync(createEventInput *v3ioc.CreateEventInput) error {
+	createEventInput.ControlPlaneInput.Timeout = 30 * time.Second
```
Comment (pavius) [inline pkg/controlplane/http/session.go]: Why tis this hard coded?

File: pkg/controlplane/types.go [hunk 3]
```diff
@@ -23,7 +23,6 @@ import (
 
 // ControlplaneSession allows operations over a controlplane session
 type Session interface {
-
```
Comment (pavius) [inline pkg/controlplane/types.go]: Reintroduce this space

Comment (FelGel) [reply pkg/controlplane/types.go]: It's removed automatically by gofmt

**Pull request merged**